### DISCUSSION TITLE
Group Promote is possible?

### DISCUSSION THREAD

**Discussion by kingbj940429** [argoproj/argo-rollouts#3629] [Q&A]
I have backend and frontend server and use blue/green with argo rollout.

currently, I create new versions of the frontend and backend as blue and manually promote them after they pass the analysis without any issues.

However, even if we promote the frontend and then immediately promote the backend, there is a momentary discrepancy. I wonder if there is a feature that allows them to be promoted simultaneously.

There is a very very short moment where the frontend is pointing to the previous version of the backend.

**Comment by kostis-codefresh** [comment]
The ideal scenario is to make the frontend a bit smarter and have a configuration parameter for which backend to use. This avoids any momentary discrepancies once and for all.

I have documented the details here https://codefresh.io/blog/multi-service-progressive-delivery-with-argo-rollouts/
